nixos/maintainers/scripts/ec2/amazon-hvm-config.nix

```diff
@@ -1,5 +1,5 @@
 { config, pkgs, ...}:
 {
-  imports = [ ./amazon-base-config.nix ];
+  imports = [ ./amazon-base-config.nix ../../../modules/testing/test-instrumentation.nix ];
```

Comment:
I only really want to stick the `test-instrumentation.nix` into the image for tests. Should I make the import conditional? Is that gross?